src/main/java/com/puppycrawl/tools/checkstyle/checks/design/DesignForExtensionCheck.java

```diff
@@ -224,6 +231,11 @@
     private Set<String> ignoredAnnotations = Arrays.stream(new String[] {"Test", "Before", "After",
         "BeforeClass", "AfterClass", }).collect(Collectors.toSet());
 
+    /**
+     * Specify the comment text pattern which qualifies as a valid javadoc for methods.
+     */
+    private Pattern requiredJavadocPhrase = Pattern.compile(".*");
```

Comment:
Yes, but since majority of the javadoc don't have this pattern, I have to create an additional property to whether check for javadoc phrases at all. Then for all I have to specify property
`javadocProperty --> off`
and to check:
```
javadocPhraseCheck --> on
requiredJavadocPhrase --> *some regex
```

Reply:
> Yes, but since majority of the javadoc don't have this pattern

Makes sense, let it be `.*` by default as you wrote initially